app/exporters/formatters/asylum_support_decision_indexable_formatter.rb

```diff
@@ -0,0 +1,23 @@
+require "formatters/abstract_specialist_document_indexable_formatter"
+
+class AsylumSupportDecisionIndexableFormatter < AbstractSpecialistDocumentIndexableFormatter
+  def type
+    "asylum_support_decision"
+  end
+
+private
+  def extra_attributes
+    {
+      tribunal_decision_decision_date: entity.tribunal_decision_decision_date,
+      tribunal_decision_judges: entity.tribunal_decision_judges,
```

Comment:
@robmckinnon 

The way to avoid having to add the `allowed_values` in rummager (discussed here:  https://github.com/alphagov/rummager/pull/502#discussion-diff-37851721), is to send the full names of the judges to rummager as well. 

The line above sends the parameterised version like `tribunal_decision_judges: ['victoria-walker']`. You would need to add something like `tribunal_decision_judge_names: ["Victoria Walker"]`. Some example voodoo to get the label for a value from the allowed_values is here: https://github.com/alphagov/specialist-publisher/commit/42a62591f474b15f51348109cae73915044d9207. I don't know if that is the "correct" way to do that.

This would enable you to use the full name value in `asylum-support-decisions.json`. Instead of having one entry do both `display_as_result_metadata: true` and `filterable: true`, you separate those out. This is an example of how to do that: https://github.com/alphagov/specialist-publisher/commit/437b89323568324aeb8212f2bd74b34ecf27f425.

I hope this makes sense.